serving/metadata-webhook/pkg/defaults/ksvc_defaults.go

```diff
@@ -10,8 +10,10 @@ import (
 const (
 	openshiftPassthrough = "serving.knative.openshift.io/enablePassthrough"
 
-	sidecarInject                = "sidecar.istio.io/inject"
-	sidecarrewriteAppHTTPProbers = "sidecar.istio.io/rewriteAppHTTPProbers"
+	sidecarInject                   = "sidecar.istio.io/inject"
+	sidecarrewriteAppHTTPProbers    = "sidecar.istio.io/rewriteAppHTTPProbers"
+	proxyIstioConfig                = "proxy.istio.io/config"
+	holdApplicationUntilProxyStarts = `{ "holdApplicationUntilProxyStarts": true }`
```

Comment:
Please let me confirm. Does this new `holdApplicationUntilProxyStarts` need for some tests?

Reply:
I've put this annotation also on the "forwarder" component. It's probably not required but doesn't harm. I just wanted to make sure istio-proxy starts in time.

Reply:
Actually, now I remember that it might be required for Forwarder as well, it's a Knative Service and the istio-proxy must start first otherwise some requests going through this component might fail during upgrade (the upgrade modifies queue-proxy so it starts a new Pod).

Reply:
Thank you! I understand!